### PR TITLE
fix: using xpty

### PR DESCRIPTION
**Pull request by caarlos0**
using x/xpty, and using term size as pty size

### CODE REVIEW

**Review by bashbunni — APPROVED** [review]
All tests pass + tried
- `-x "git status"`
- `main.go -o main.png` 
- `main.go -o main.svg`

all worked as expected

File: go.mod [hunk 1]
```diff
@@ -16,7 +16,8 @@ require (
 	github.com/charmbracelet/log v0.4.0
 	github.com/charmbracelet/x/ansi v0.4.0
 	github.com/charmbracelet/x/exp/term v0.0.0-20240403043919-dea9035a27d4
-	github.com/creack/pty v1.1.23
+	github.com/charmbracelet/x/term v0.2.0
+	github.com/charmbracelet/x/xpty v0.1.0
```
Comment (aymanbagabas) [inline go.mod:20]:
```suggestion
	github.com/charmbracelet/x/xpty v0.1.1
```

Comment (aymanbagabas) [reply go.mod:20]: We need to update xpty

**Review by aymanbagabas — COMMENTED** [review]
Looks good otherwise

**Comment by bashbunni** [comment]
Need to update deps, then good to merge :)

**Pull request merged**